src/components/dropdown-disclosure/components/list-item/index.tsx

```diff
@@ -97,10 +97,10 @@ const DropdownDisclosureButtonItem = ({
 					toggleDisclosure()
 				}}
 			>
-				{icon}
 				<Text asElement="span" size={200} weight="medium">
 					{children}
 				</Text>
+				{icon}
```

Comment:
Need to validate whether I can globally change this icon placement. I _think_ its okay.

Reply:
Ah yes, these two places are where we want the icon to flip so thats good!